src/Sfa.Das.Sas.Infrastructure/Shared/Elasticsearch/ElasticsearchClientFactory.cs

```diff
@@ -35,7 +36,10 @@ public IElasticClient GetElasticClient()
                     new SingleNodeConnectionPool(_infrastructureSettings.ElasticServerUrls.FirstOrDefault()));
             }
 
-            settings.BasicAuthentication(_infrastructureSettings.ElasticsearchUsername, _infrastructureSettings.ElasticsearchPassword);
+            if (!Debugger.IsAttached)
```

Comment:
Did you intend to leave this in? Or was it for development purposes only

Reply:
Intended